components/experiment-creation/ExperimentForm.tsx

```diff
@@ -0,0 +1,45 @@
+// Temporarily ignore until more parts are in place
+/* eslint-disable */
```

Comment:
It'd be better to make it more specific:

```suggestion
/* eslint-disable @typescript-eslint/no-unused-vars */
```